tools/tsp-client/src/index.ts

```diff
@@ -204,6 +204,10 @@ const parser = yargs(hideBin(process.argv))
         .option("fully-compatible", {
           type: "boolean",
           description: "Convert swagger to fully compatible TypeSpec",
+        })
+        .option("debug", {
+          type: "boolean",
+          description: "Enable debug logging",
```

Comment:
We dont need this, there's already a global debug option for you to check.

Reply:
Just going to clarify here as well even though you need to remove this option here, you can still access the global `--debug` flag defined earlier in this file with `const debug = argv["debug"];`